contracts/src/SemiFungible1155.sol

```diff
@@ -419,18 +411,23 @@ contract SemiFungible1155 is Upgradeable1155 {
         return array;
     }
 
+    // UUPS PROXY
+
+    /// @dev see { openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol }
+    function _authorizeUpgrade(address newImplementation) internal virtual override onlyOwner {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
     /**
      * @dev This empty reserved space is put in place to allow future versions to add new
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      * Assuming 30 available slots (slots cost space, cost gas)
      * 1. typeCounter
-     * 2. TYPE_MASK
-     * 3. NF_INDEX_MASK
-     * 4. owners
-     * 5. creators
-     * 6. tokenValues
-     * 7. maxIndex
+     * 2. owners
+     * 3. creators
+     * 4. tokenValues
+     * 5. maxIndex
      */
-    uint256[23] private __gap;
+    uint256[25] private __gap;
```

Comment:
Hmmm... I'm deducing that constants don't count towards storage?
So we're adding it back into __gap?

But previously this was set to 23, and without actually reclaiming any storage, this upgrade will break right? Just wanted to check that I understood correctly,

Reply:
Yes, but changes are not really breaking because the `gap` is at the end.